components/FeatureHighlight/src/private/MDCFeatureHighlightView.m

```diff
@@ -76,6 +78,7 @@ - (instancetype)initWithFrame:(CGRect)frame {
     _titleLabel.textAlignment = NSTextAlignmentNatural;
     _titleLabel.lineBreakMode = NSLineBreakByTruncatingTail;
     _titleLabel.numberOfLines = 0;
+    _titleLabel.accessibilityHint = NSLocalizedString(MDCFeatureHighlightViewTitleA11yHint, nil);
```

Comment:
This will fail as it will try and load from the mainBundle, see MDCPageControl for an example of how to load from a local .bundle.